src/CoreFx.Private.TestUtilities/ref/CoreFx.Private.TestUtilities.cs

```diff
@@ -82,11 +82,9 @@ public static partial class PlatformDetection
         public static bool IsUbuntu1704 { get { throw null; } }
         public static bool IsUbuntu1710 { get { throw null; } }
         public static bool IsWindows { get { throw null; } }
-        public static bool IsWindows10InsiderPreviewBuild16215OrGreater { get { throw null; } }
         public static bool IsWindows10Version1607OrGreater { get { throw null; } }
-        public static bool IsWindows10Version16251OrGreater { get { throw null; } }
         public static bool IsWindows10Version1703OrGreater { get { throw null; } }
-        public static bool IsWindowsRedStone2 { get { throw null; } }
+        public static bool IsWindows10Version1709OrGreater { get { throw null; } }
```

Comment:
It might be nice to have a comment on each of these giving their human readable product names..

Reply:
I'll add the official marketing terms "Anniversay Update", "Creators Update", etc. comments to these.  We don't, though, use the codename (RS) terminology.